runtime/local/src/lib.rs

```diff
@@ -544,6 +547,173 @@ impl pallet_custom_signatures::Config for Runtime {
     type UnsignedPriority = EcdsaUnsignedPriority;
 }
 
+parameter_types! {
+    pub MaximumSchedulerWeight: Weight = NORMAL_DISPATCH_RATIO * RuntimeBlockWeights::get().max_block;
+    pub const MaxScheduledPerBlock: u32 = 50;
+}
+
+impl pallet_scheduler::Config for Runtime {
+    type Event = Event;
+    type Origin = Origin;
+    type PalletsOrigin = OriginCaller;
+    type Call = Call;
+    type MaximumWeight = MaximumSchedulerWeight;
+    type ScheduleOrigin = frame_system::EnsureRoot<AccountId>;
+    type MaxScheduledPerBlock = MaxScheduledPerBlock;
+    type WeightInfo = pallet_scheduler::weights::SubstrateWeight<Runtime>;
+    type OriginPrivilegeCmp = EqualPrivilegeOnly;
+    type PreimageProvider = ();
+    type NoPreimagePostponement = ();
+}
+
+parameter_types! {
+    pub CouncilMotionDuration: BlockNumber = 3 * MINUTES;
+    pub const CouncilMaxProposals: u32 = 100;
+    pub const CouncilMaxMembers: u32 = 100;
```

Comment:
Perhaps a smaller amount would be more practical for testing? E.g. `3`?

Reply:
`CouncilMaxMembers` 5 would be better. because we have a threshold configuration 60% (3/5 council members).
For example at pallet_treasury `ApproveOrigin`
```
pallet_collective::EnsureProportionAtLeast<AccountId, CouncilCollective, 3, 5>,
```

Reply:
Sure, that also works.
We could also change it to 2/3 though.

Reply:
yes, that's also true.
For local node, both is acceptable. I go with 2/3.